### PR TITLE
add trait with multi-column support with tests

### PR DESCRIPTION
**Pull request by davitbek**

### CODE REVIEW

**Comment by davitbek** [comment]
Hi All. Can you review my PR please

File: src/SchemalessAttributesTrait.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+/**
+ * @property array $schemalessAttributes
+ *
+ * @mixin Collection
+ */
+trait SchemalessAttributesTrait
+{
+    /**
+     * Register schemalessAttributes attributes as array(cast).
```
Comment (freekmurze) [inline src/SchemalessAttributesTrait.php:15]: Remove this comment

File: src/SchemalessAttributesTrait.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        }
+    }
+
+    /**
+     * @return array
+     */
+    public function getSchemalessAttributes()
+    {
+        return $this->schemalessAttributes ?? [];
+    }
```
Comment (freekmurze) [inline src/SchemalessAttributesTrait.php:30]: Remove the docblock in favour of typehints

**Review by freekmurze — CHANGES_REQUESTED** [review]

**Comment by davitbek** [comment]
Hi @freekmurze I fixed unit tests, and make your suggested changes. Can you check please?

File: src/SchemalessAttributesTrait.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    private $initializeSchemalessAttributesTrait;
+
+    /**
+     * Get the casts array.
+     *
+     * @return array
+     */
+    public function getCasts()
+    {
+        if (Str::startsWith(app()->version(), '5.6')) {
```
Comment (freekmurze) [inline src/SchemalessAttributesTrait.php:27]: Instead of doing this check, let's just drop support for Laravel 5 allthogheter

**Review by freekmurze — CHANGES_REQUESTED** [review]
One more change needed to simplify things

File: src/SchemalessAttributesTrait.php (hunk 3 above)
Comment (davitbek) [reply src/SchemalessAttributesTrait.php]: @freekmurze Done. Please check

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**